README.md

```diff
@@ -4,6 +4,13 @@
 
 This is a plugin for Foreman that introduces integration with [NetBox](https://netbox.readthedocs.io)
 
+## Compatibility
+
+| Netbox Version | Plugin Version |
+| -------------- | -------------- |
+| 2.8            | 1.0            |
+| 2.11           | 1.1            |
```

Comment:
@danielwieberneit I think we should release `1.1` as there are some breaking changes

Reply:
release 1.1 is fine, but we were on 1.0.2 (Netbox 2.8)

Reply:
I believe we can ignore the patch version in the README and just leave `1.0` as there should be no breaking changes between `1.0.0`, `1.0.1` and `1.0.2`. Version `1.1.0` won't be compatible with Netbox < `2.11`